api/Containerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8/python-39
 
 WORKDIR /usr/src/app
```

Comment:
add `USER root` under here to support non-root containerized environments. Podman in particular runs in user space and npm is finicky about the permissions